PPVIS_2/View/AuthorizationWindow.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  AuthorizationWindow.swift
+//  ppvis-2
+//
+//  Created by Алина Ненашева on 2.12.20.
+//
+
+import UIKit
+
+class AuthorizationWindow: UIViewController {
+    
+    @IBOutlet weak var programLabel: UILabel!
+    @IBOutlet weak var loginTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var submitButton: UIButton!
+    @IBOutlet weak var loginLabel: UILabel!
+    @IBOutlet weak var passwordLabel: UILabel!
+    let mainMenu = MainMenu()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        passwordTextField.isSecureTextEntry = true
+    }
+    
+    func getPasswordTextField()-> String {
+        return passwordTextField.text ?? ""
+    }
+    
+    func getLoginTextField()-> String {
+        return loginTextField.text ?? ""
+    }
+    
+    func getSubmitButton()-> UIButton {
+        return submitButton
+    }
+    
```

Comment:
Для того чтоб не раскидывать кому попало UI элементы можно сделать следующим образом:
```suggestion
    private var submitButtonHandler: (() -> ())?
    
    func setSubmitButtonHandler(_ handler: @escaping () -> ()) {
        submitButtonHandler = handler
    }
    
    @IBAction func submitButtonPressed(_ sender: Any) {
        submitButtonHandler?()
    }
    
```
И тогда в месте где задается обработчик будет не authorizationWindow.getSubmitButton().addTarget(self, action: #selector(verifyPasswordForEntrance), for: .touchUpInside), а authorizationWindow.setSubmitButtonHandler(verifyPasswordForEntrance)